dependency-check-suppressions.xml

```diff
@@ -1,8 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Reset this file once hmcts/ccd-data-store-api#1831 is brought up to match `master` branch.